packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts

```diff
@@ -15,20 +17,20 @@ export async function uptimeRobotApiRequest(this: IExecuteFunctions | IExecuteSi
 	let options: OptionsWithUri = {
 		method,
 		qs,
-		body,
 		form: {
 			api_key: credentials.apiKey,
 			...body,
 		},
-		uri: uri ||`https://api.uptimerobot.com/v2${resource}`,
+		uri: uri || `https://api.uptimerobot.com/v2${resource}`,
 		json: true,
 	};
 	options = Object.assign({}, options, option);
-	if (Object.keys(options.body).length === 0) {
-		delete options.body;
-	}
 	try {
-		return await this.helpers.request!(options);
+		const responseData = await this.helpers.request(options);
+		if (responseData.stat !== 'ok') {
+			throw new NodeApiError(this.getNode(),responseData.error);
```

Comment:
I would think `NodeOperationError` is preferable for these cases, but there are examples of both in the codebase - see [Slack](https://github.com/n8n-io/n8n/blob/5c65a73cc21ac4348eaae6b6b010980ff789cefe/packages/nodes-base/nodes/Slack/GenericFunctions.ts#L60-L62) vs. [ActiveCampaign](https://github.com/n8n-io/n8n/blob/4801510b0cbc86390f8e3ca8eef008c2880f14ec/packages/nodes-base/nodes/ActiveCampaign/GenericFunctions.ts#L55-L57).

Thoughts welcome @BHesseldieck 

Also, no need to specify the `error` property in `responseData.error` - it should be found.